src/ios/GooglePlus.h

```diff
@@ -1,16 +1,13 @@
 #import <Cordova/CDVPlugin.h>
 #import <GoogleSignIn/GoogleSignIn.h>
 
-@interface GooglePlus : CDVPlugin<GIDSignInDelegate, GIDSignInDelegate>
+@interface GooglePlus : CDVPlugin
 
 @property (nonatomic, copy) NSString* callbackId;
 @property (nonatomic, assign) BOOL isSigningIn;
 
-- (void) isAvailable:(CDVInvokedUrlCommand*)command;
 - (void) login:(CDVInvokedUrlCommand*)command;
-- (void) trySilentLogin:(CDVInvokedUrlCommand*)command;
 - (void) logout:(CDVInvokedUrlCommand*)command;
 - (void) disconnect:(CDVInvokedUrlCommand*)command;
-- (void) share_unused:(CDVInvokedUrlCommand*)command;
```

Comment:
Compatibility breakage in iOS:

- `isAvailable`: even necessary?
- `trySilentLogin`: open to backfills if community still requires it
- `share_unused`: seemed to be cruft